scripts/Provision-M365DataCollector.ps1

```diff
@@ -149,17 +149,27 @@ param
     # Office Domain name
     [ValidateScript( {$_ -notmatch 'https://+' -and $_ -notmatch 'http://+'})]
     [Parameter(Mandatory = $False,
-        HelpMessage = "Office 365 Domain Name: ",
+        HelpMessage = "Office 365 Domain Name",
         Position = 10
     )]
     [ValidateNotNullOrEmpty()]
     [string]
     $OfficeDomain = $(Read-Host -prompt "Enter Office 365 Domain Name"),
 
+    # Office Domain URL
+    [ValidateScript( {$_ -notmatch 'https://+' -and $_ -notmatch 'http://+'})]
+    [Parameter(Mandatory = $False,
+        HelpMessage = "Share Point Domain",
+        Position = 11
+    )]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $SharePointDomain = $(Read-Host -prompt "Enter Share Point Domain"),
```

Comment:
convert **SharepointDomainURL** to **SharePointAdminCenterURL** as SharePoint domain URL is misleading

Reply:
done